pkg/mutt

```diff
@@ -16,7 +16,7 @@ cyrus-sasl
 perl
 
 [build]
-patch -p1 < "$K"/mutt-netbsd-curses.patch
+#patch -p1 < "$K"/mutt-netbsd-curses.patch
```

Comment:
commented out - not needed, or did you install ncurses over curses ?
if not needed remove the line and the patch